scripts/functions/diagnostic_conda.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#* This file is part of the MOOSE framework
+#* https://www.mooseframework.org
+#*
+#* All rights reserved, see COPYRIGHT for full restrictions
+#* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+#*
+#* Licensed under LGPL 2.1, please see LICENSE for details
+#* https://www.gnu.org/licenses/lgpl-2.1.html
+
+function conda_url()
+{
+    local CONDA_URL='https://github.com/conda-forge/miniforge/releases/download'
+    echo ${CONDA_URL}/${CONDA_VERSION}/${INSTANCE_SUP}-${CONDA_VERSION}
+}
+
+function create_env()
+{
+    if [ "${PRISTINE_ENVIRONMENT}" == 0 ]; then return; fi
+    printf "Creating MOOSE Conda Environment...\n"
+    if [ ${FULL_BUILD} == 0 ]; then MOOSE_PACKAGES='moose-dev'; fi
+    export CONDARC=$CTMP_DIR/.condarc
+    export CONDA_ENVS_PATH=$CTMP_DIR/_env/.envs
+    ${INSTANCE_EXE} config --env --set ssl_verify false
```

Comment:
same can we not use this in our first try? maybe we dont care if we are going to use it later

Reply:
should we force the user to pass --insecure to the diagnostics?

Reply:
I should be able to get rid of all insecure occurrences.

These are here because I didn't know all the influential environment variables needed to be passed into my pristine environment in order to do this securely.